root/etc/e-smith/templates/etc/opt/rh/rh-php71/php-fpm.d/z_nethserver.conf/10base

```diff
@@ -0,0 +1,2 @@
+[www]
+listen = 127.0.0.1:9001
```

Comment:
You need to set the port inside the db [prop](https://github.com/NethServer/nethserver-rh-php71-php-fpm/blob/master/root/etc/e-smith/db/configuration/defaults/rh-php71-php-fpm/TCPPorts), then read the value and apply it here.

Reply:
done